docs/_layouts/documentation.html

```diff
@@ -27,5 +27,6 @@
   </div>
 </div>
 
+<style>body{overflow: hidden;}</style>
```

Comment:
We should not need that `overflow:hidden` rule (especially inlined :p) -> we can probably find the guilty div and fix it.